pages/common/git-whatchanged.md

```diff
@@ -0,0 +1,17 @@
+# git whatchanged
+
+> Show what has changed with recent commits or files.
+> See also `git log`.
+> More information <https://git-scm.com/docs/git-whatchanged>.
+
+- Display log of recent commits:
+
+`git whatchanged`
+
+- Display log of recent commits within the specified time frame:
+
+`git whatchanged --since="2 hours ago"`
```

Comment:
```suggestion
`git whatchanged --since="{{2 hours ago}}"`
```

Reply:
Not sure about this

Reply:
We normally do that whenever the thing is user-configurable (e.g. a variable or literal).  It also helps distinguish flags from values, etc., as @marchersimon pointed out [here](https://github.com/tldr-pages/tldr/pull/5921#issuecomment-836462426).